app/views/unattended/provisioning_templates/snippet/kickstart_rhsm.erb

```diff
@@ -14,16 +32,25 @@ test_on:
   subman_registration = host_param_true?('subscription_manager') || subman_keys.present?
   subman_hostname = " --server-hostname #{@host.content_source.rhsm_url.host}" if (plugin_present?('katello') && @host.content_source)
   subman_rhsm_baseurl = " --rhsm-baseurl #{@host.content_source.pulp_content_url}" if (plugin_present?('katello') && @host.content_source)
+  subman_hostname = " --server-hostname #{host_param('rhsm_host')}" if host_param('rhsm_host')
```

Comment:
When I create host I select content source for it, why to override it with host parameters, what is the use case?

Reply:
Without Katello installed you don't have access to select the content source.

In our use case we keep katello/satellite installed for content management but use foreman for host provisioning and puppet ENC.

Reply:
Interesting. Why don't you integrate it in a single installation?

Reply:
Mostly from previous 10 years of experience. Early Satellite/Capsule installs would break every they were upgraded. I also noticed a similar issue with Katello in a previous job. 

We have found it to be easier to separate the concerns and keep puppet/foreman working without katello integrations. Scaling Foreman with multiple instances as an ENC was easier than Satellite last I checked. 

With them separate satellite / katello can be down for days when we have to troubleshoot a issue upgrading. Although this has gotten less frequent in recent years it still happens. 

Puppet still runs without any issues under this setup.

Reply:
@ekohl Does this answer your question? Do you have any other questions?